### PR TITLE
reconfig packaging for 2.4

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: repoclosure/yum.conf [hunk 1]
```diff
@@ -98,52 +98,52 @@ name=el7-puppet-6
 baseurl=https://yum.puppetlabs.com/puppet6/el/7/x86_64/
 
 # Used as lookaside repos for layered repos (plugins)
-[el7-foreman-nightly]
-name=Foreman nightly EL7
-baseurl=http://koji.katello.org/releases/yum/foreman-nightly/RHEL/7/x86_64/
+[el7-foreman-2.4]
+name=Foreman 2.4 EL7
+baseurl=http://koji.katello.org/releases/yum/foreman-4.0/RHEL/7/x86_64/
```
Comment (ekohl) [inline repoclosure/yum.conf:103]: This and other base urls should be 2.4.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: package_manifest.yaml [hunk 2]
```diff
@@ -786,7 +730,7 @@ katello_packages:
           foremandist: "{{ foremandist }}"
     releasers:
       - koji-katello
-    nightly_releaser: koji-katello-jenkins
+    4.0_releaser: koji-katello-jenkins
```
[the remark this answers is not in the document]
Comment (evgeni) [reply package_manifest.yaml]: no, missed when deleting

File: repoclosure/yum.conf (hunk 1 above)
Comment (evgeni) [reply repoclosure/yum.conf]: right, regex gone wrong :see_no_evil:

File: repoclosure/yum.conf [hunk 3]
```diff
@@ -98,52 +98,52 @@ name=el7-puppet-6
...
 [el7-configmanagement-salt]
 name=SaltStack configmanagement
 baseurl=https://repo.saltstack.com/py3/redhat/7/x86_64/latest
 
 # Katello repos for lookaside
-[el7-katello-nightly]
-name=Katello nightly EL7
-baseurl=http://koji.katello.org/releases/yum/katello-nightly/katello/el7/x86_64/
+[el7-katello-4.0]
+name=Katello 2.4 EL7
```
Comment (ekohl) [inline repoclosure/yum.conf:129]:
```suggestion
name=Katello 4.0 EL7
```

File: repoclosure/yum.conf [hunk 4]
```diff
@@ -98,52 +98,52 @@ name=el7-puppet-6
...
-name=Katello nightly EL8
-baseurl=http://koji.katello.org/releases/yum/katello-nightly/katello/el8/x86_64/
+[el8-katello-4.0]
+name=Katello 2.4 EL8
+baseurl=http://koji.katello.org/releases/yum/katello-4.0/katello/el8/x86_64/
 
-[el7-katello-pulp-nightly]
-name=Katello Pulp nightly EL7
+[el7-katello-pulp-4.0]
+name=Katello Pulp 2.4 EL7
```
Comment (ekohl) [inline repoclosure/yum.conf:137]: Should this be 2.19 or 4.0?

**Review by ekohl — CHANGES_REQUESTED** [review]
It would be nice if we could say `Katello for Foreman 2.4` to avoid the exact Katello numbering. Maybe a good goal for 2.5/4.1?

File: repoclosure/yum.conf (hunk 4 above)
Comment (evgeni) [reply repoclosure/yum.conf]: 4.0,

**Review by ekohl — APPROVED** [review]

**Pull request merged**